### PR TITLE
PvC example using ansible-navigator and collection playbooks

### PR DESCRIPTION
**Pull request by jimright**
Example of CDP Private Cloud deployment using v2 of Cloudera-Deploy.

### CODE REVIEW

File: private-cloud/ecs-on-aws/pre_setup.yml [hunk 1]
```diff
@@ -90,7 +90,8 @@
   tasks:
     - name: Set up the FreeIPA server
       ansible.builtin.import_role:
-        name: cloudera.exe.freeipa_server
+        # name: cloudera.exe.freeipa_server
+        name: cloudera.infra.freeipa_server
```
Comment (wmudge) [inline private-cloud/ecs-on-aws/pre_setup.yml:94]: For now, at least?

**Comment by cmperro** [comment]
Execution Observations:

File: private-cloud/ecs-on-aws/README.md [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+```bash
+ansible-navigator run pre_setup.yml \
+-e @config.yml \
+-e @definition.yml
+```
+
+Once the pre-setup playbook completes confirm that:
+
+* You can connect to each node via the inventory - see Confirm SSH Connectivity. Note that a A `validate_dns_lookups.yml` Playbook exists to check connectivity.
+* Connect to FreeIPA UI and login with the `IPA_USER` and `IPA_PASSWORD` credentials in the configuration file. See Cluster Access for more details.
```
Comment (cmperro) [inline private-cloud/ecs-on-aws/README.md:114]: How does one do this? My FreeIPA node just has port 22 open, to the world, I might add...

File: private-cloud/ecs-on-aws/README.md [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+    | `AWS_PROFILE` | The profile label for your AWS credentials. Otherwise, use the associated `AWS_*` parameters. Used also for remote storage of Terraform state in AWS. | Mandatory |
+
+> **_NOTE:_** For OSX, set the following: `export OBJC_DISABLE_INITIALIZE_FORK_SAFETY=YES` to allow the WinRM modules to function.
+
+
+#### Configuration file variables
+
+Edit the `config.yml` user-facing configuration file to match your particular deployment.
+
+*NOTE:* `name_prefix` should be 4-8 characters and is the "primary key" for the deployment. `owner_prefix` is used in circumstances to differentiate resources such as the SSH key label in the cloud provider and the subdomain(s) for the private DNS service.
```
[the remark this answers is not in the document]
Comment (jimright) [reply private-cloud/ecs-on-aws/README.md]: Updated to readme with this constraint.

**Comment by jimright** [comment]
Closing. @wmudge will open new PR with updated infra deployment.

**Pull request closed without merging**